Capfile

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Load DSL and set up stages
+require 'capistrano/setup'
+
+# Include default deployment tasks
+require 'capistrano/deploy'
+
+require 'capistrano/scm/git'
+install_plugin Capistrano::SCM::Git
+
+require 'capistrano/bundler'
+require 'capistrano/rails'
+require 'capistrano/honeybadger'
+require 'capistrano/passenger'
+require 'capistrano/maintenance'
```

Comment:
Do we need to do more for maintenance to work correctly?

Reply:
Maybe? I can add more to this PR, I was focused so far on just getting the deployment done. But happy to add that here or in a follow up.

Reply:
@justinlittman I don't believe so, The PR when you added to H2 was just the gem (https://github.com/sul-dlss/happy-heron/commit/6c5fc9835cbee95efc42bdece6f49765a44fe4e5) and looking at the readme (https://github.com/capistrano/maintenance) I think we might just have some template work to do when we actually want to put up a maintenance page, but I think that can happen later.

Reply:
Should we yank out the maintenance stuff and add a backlog ticket to set up maintenance for real?